netman/adapters/switches/brocade.py

```diff
@@ -478,6 +471,14 @@ def parse_if_ranges(string):
         consumed_string = consumed_string[len(parsed_part):].strip()
 
 
+def _to_real_names(if_list):
+    return [i.replace("ethe", "ethernet") for i in if_list]
+
```

Comment:
Is this case possible ?

>>> def _to_real_names(if_list):
...     return [i.replace("ethe", "ethernet") for i in if_list]
...
>>> _to_real_names(['ethe 1/2/2'])
['ethernet 1/2/2']
>>> _to_real_names(['ethernet 1/2/2'])
['ethernetrnet 1/2/2']